tests/Vendor/Order/OrderServiceTest.php

```diff
@@ -77,27 +77,35 @@ public function testListOrders(): void
         $orders = $this->buildVendorOrderService()->listOrders();
 
         $this->assertContainsOnly(OrderSummary::class, $orders);
-        $this->assertCount(2, $orders);
+        $this->assertTrue(count($orders) >= 2);
```

Comment:
pourquoi ce test est moins précis qu'avant ? on doit toujours avoir le même nombre d'orders non ?
```suggestion
        static::assertTrue(count($orders) >= 2);
```

Reply:
Non, car c'est basé sur les fixture de l'environnement de wizaplace de dev, qui peuvent évoluer. Du coup quand on regénére les cassettes 6 mois plus tard, le nombre d'order à peut-être changé.